node/network/bitfield-distribution/src/lib.rs

```diff
@@ -605,6 +621,53 @@ where
 	}
 }
 
+#[derive(Clone)]
+struct MetricsInner {
+	gossipped_own_availability_bitfields: prometheus::Counter<prometheus::U64>,
```

Comment:
Not 100% sure about the prometheus API, but there should `Gauge`s which might be more suitable?

Reply:
`Gauge` is allowed to go down as well, here we count the total number of bitfields gossipped over time. Prometheus has query language to ask what would that be in a specific time range.

Any specific reason you think it might be more suitable?